Sources/SwiftDocC/Infrastructure/DocumentationConverter.swift

```diff
@@ -262,7 +262,7 @@ public struct DocumentationConverter: DocumentationConverterProtocol {
         try outputConsumer.consume(assetsInBundle: bundle)
         
         let symbolIdentifiersMeetingRequirementsForExpandedDocumentation: [String]? = symbolIdentifiersWithExpandedDocumentation?.compactMap { (identifier, expandedDocsRequirement) -> String? in
-            guard let documentationNode =  context.nodeWithSymbolIdentifier(identifier) else {
+            guard let documentationNode = context.documentationCache[identifier] else {
```

Comment:
Which is the difference of accessing the context nodes's directly vs the cache? The cache contains external + internal symbols while the context only internal?

Reply:
There is no difference. `nodeWithSymbolIdentifier(_:)` used to be implemented like below, as a way of accessing a node by its symbol identifier. 

```swift
func nodeWithSymbolIdentifier(_ preciseIdentifier: String) -> DocumentationNode? {
    guard let reference = symbolIndex[preciseIdentifier] else { return nil }
    return documentationCache[reference]
}
```

Since the new documentation cache has subscripts for both resolved references and for symbol identifiers, there's no longer any need for `nodeWithSymbolIdentifier(_:)`. The updated implementation would just be 
```swift
func nodeWithSymbolIdentifier(_ preciseIdentifier: String) -> DocumentationNode? {
    return documentationCache[preciseIdentifier]
}
```

which is just a worse version of calling `documentationCache[preciseIdentifier]` directly because it doesn't have the same ability to optimize away copying the accessed value.